aten/src/ATen/native/native_functions.yaml

```diff
@@ -5002,6 +5002,17 @@
   device_check: NoCheck
   device_guard: False
 
+
+- func: squeeze.dims(Tensor(a) self, int[] dim) -> Tensor(a)
```

Comment:
What is the reason for not re-using the `squeeze.dim` overload above with `int[1]` as the new argument?
Just avoid BC issues?

Reply:
I tried in #91603 but it breaks BC so I think we need to keep this.

Reply:
Ho sad, ok !